### PR TITLE
Updates RpcConstants to have appropriate ViewDescriptors.

### PR DESCRIPTION
**Pull request by dinooliva**

### CODE REVIEW

File: core/java/com/google/instrumentation/stats/RpcConstants.java [hunk 1]
```diff
@@ -60,6 +60,11 @@
           "/rpc/client/roundtrip_latency",
           "RPC roundtrip latency us",
           MeasurementUnit.create(-6, scalar));
+  public static final MeasurementDescriptor RPC_CLIENT_SERVER_ELAPSED_TIME =
+      MeasurementDescriptor.create(
+          "/rpc/client/server_elapsed_time",
+          "Server elapsed time in msecs",
+          MeasurementUnit.create(-3, scalar));
```
Comment (a-veitch) [inline core/java/com/google/instrumentation/stats/RpcConstants.java]: Shouldn't this be seconds, not scalar?

File: core/java/com/google/instrumentation/stats/RpcConstants.java [hunk 2]
```diff
@@ -89,6 +94,11 @@
           "/rpc/server/response_bytes",
           "Response MB/s",
           MeasurementUnit.create(6, bytes, seconds));
+  public static final MeasurementDescriptor RPC_SERVER_SERVER_ELAPSED_TIME =
+      MeasurementDescriptor.create(
+          "/rpc/server/server_elapsed_time",
+          "Server elapsed time in msecs",
+          MeasurementUnit.create(-3, scalar));
```
Comment (a-veitch) [inline core/java/com/google/instrumentation/stats/RpcConstants.java]: s/scalar/seconds

File: core/java/com/google/instrumentation/stats/RpcConstants.java [hunk 3]
```diff
@@ -105,6 +115,108 @@
           "Uncompressed Request MB/s",
           MeasurementUnit.create(6, bytes, seconds));
 
+  // Common sized histogram bucket boundaries.
+  private static final List<Double> bytes_bucket_boundaries = Arrays.asList(
```
Comment (a-veitch) [inline core/java/com/google/instrumentation/stats/RpcConstants.java]: Being constants, should these be BYTES_BUCKET_BOUNDARIES ?

File: core/java/com/google/instrumentation/stats/RpcConstants.java (hunk 1 above)
Comment (dinooliva) [inline core/java/com/google/instrumentation/stats/RpcConstants.java]: Yes - done.

File: core/java/com/google/instrumentation/stats/RpcConstants.java (hunk 3 above)
Comment (dinooliva) [inline core/java/com/google/instrumentation/stats/RpcConstants.java]: Done.

File: core/java/com/google/instrumentation/stats/RpcConstants.java [hunk 4]
```diff
@@ -82,28 +87,136 @@
...
   public static final MeasurementDescriptor RPC_SERVER_UNCOMPRESSED_RESPONSE_BYTES =
       MeasurementDescriptor.create(
           "/rpc/server/uncompressed_response_bytes",
-          "Uncompressed Request MB/s",
-          MeasurementUnit.create(6, bytes, seconds));
+          "Uncompressed Request MB",
+          MeasurementUnit.create(6, bytes));
+
+  // Common histogram bucket boundaries for bytes received/sets DistributionViewDescriptors.
+  private static final List<Double> BYTES_BUCKET_BOUNDARIES = Arrays.asList(
```
Comment (a-veitch) [inline core/java/com/google/instrumentation/stats/RpcConstants.java]: maybe RPC_BYTES_BUCKET_BOUNDARIES?

File: core/java/com/google/instrumentation/stats/RpcConstants.java [hunk 5]
```diff
@@ -82,28 +87,136 @@
...
   public static final MeasurementDescriptor RPC_SERVER_UNCOMPRESSED_RESPONSE_BYTES =
       MeasurementDescriptor.create(
           "/rpc/server/uncompressed_response_bytes",
-          "Uncompressed Request MB/s",
-          MeasurementUnit.create(6, bytes, seconds));
+          "Uncompressed Request MB",
+          MeasurementUnit.create(6, bytes));
+
+  // Common histogram bucket boundaries for bytes received/sets DistributionViewDescriptors.
+  private static final List<Double> RPC_BYTES_BUCKET_BOUNDARIES = Arrays.asList(
```
Comment (adriancole) [inline core/java/com/google/instrumentation/stats/RpcConstants.java]: these look legit, just some notion of where they were derived might help

**Pull request merged**